submissions/zalevskyi/html-movie-seat-booking/app.css

```diff
@@ -0,0 +1,112 @@
+* {
+    background: whitesmoke;
+    margin: 0;
+}
+.title {
+    padding: 10px;
+    text-align: center;
+    background: lightsalmon;
+    font-weight: normal;
+    font-size: 1.5rem;
+}
+.film__title, .film__date, .film__time {
+    width: 75vw;
+    margin: 10px auto;
+    text-align: left;
+}
+.seats_legend__wrapper {
+    margin-top: 20px;
+    text-align: center;
+}
+.description {
+    font-size: 1em;
+    line-height: 1em;
+}
+.screen {
+    width: 75vw;
+    margin: 20px auto 5vh auto;
+    box-shadow: 0px 5px 0px lightskyblue;
+    text-transform: uppercase;
+    text-align: center;
+}
+.seat {
```

Comment:
Add styles on `:hover`, `:active`, and `:focus` states. Also, add `cursor: pointer;`